src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationNewHandler.java

```diff
@@ -20,4 +20,11 @@ public void test() { //indent:4 exp:4
         } catch (Exception e) { //indent:8 exp:8
         } //indent:8 exp:8
     } //indent:4 exp:4
+
+    int[][] array10b //indent:4 exp:4
+= new int[][] { //indent:0 exp:8 warn
```

Comment:
it is very good to use case from issue.

but please use values from regression report like:
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b8cb762_2021143007/reports/diff/android-launcher/index.html#A1
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b8cb762_2021143007/reports/diff/android-launcher/index.html#A2
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b8cb762_2021143007/reports/diff/apache-ant/index.html#A1

we should not reply on diff report to catch specific type of code structure.
We should see same diff in UT validation, diff report will same as UT but with huge amount of violations of the same type.

Reply:
Thank you for your review! I added more code to the test resources. I also added method `additionalOffsets` to `LineWrappingHandler` to accept https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b8cb762_2021143007/reports/diff/apache-ant/index.html#A1 which is in my opinion correctly indented.

Reply:
to make it valid (no violation) forceStrictCondition should FALSE.

with forceStrictCondition is TRUE - there should be violation.

Reply:
I do not see in Inputs case like 
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b8cb762_2021143007/reports/diff/android-launcher/index.html#A2

please add.

Reply:
See new commit https://github.com/checkstyle/checkstyle/commit/6d04f1645f82fadbdf5496d2b60d38810612e575